compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -386,8 +387,8 @@ setConvolutionVectorDistributionConfig(IREE::GPU::TargetAttr target,
   // for later access in the pipeline.
   SmallVector<NamedAttribute, 1> pipelineAttrs;
   auto scheduleAttr = IREE::GPU::MMAScheduleAttr::get(
-      context, target.getWgp().getMma()[schedule->index], schedule->mWarpCount,
-      schedule->nWarpCount);
+      context, target.getWgp().getMma()[schedule->index],
+      schedule->mSubgroupCounts[0], schedule->nSubgroupCounts[0]);
```

Comment:
Can you add a TODO here, to take all subgroup counts into account? Easier to keep track that way.